dashboard/test/ui/features/applab/scenarios.feature

```diff
@@ -37,3 +37,15 @@ Feature: App Lab Share
     And element "#codeModeButton" is hidden
     And element "#designModeButton" is hidden
     And element "#viewDataButton" is hidden
+
+  @no_mobile
+  Scenario: App Lab Http Image
+
+    # Create an app with an http image.
+    Then I press "show-code-header"
+    And I wait to see Droplet text mode
+    And I press keys "image('test123', 'http://example.com" for element ".ace_text-input"
+    And I press "runButton"
+    And I wait until element "#divApplab > .screen > img#test123" is visible
+    And element "#divApplab > .screen > img#test123[src=\'https://studio.code.org/media?u=http%3A%2F%2Fexample.com\']" is visible
```

Comment:
This feels a little bit awkward. I'm assuming what we care about is not that this element is visible, but that it does in fact have this src value?. I wonder if we should create a new step that is something along the lines of:
And element "selector" has attribute "attribute_name" equal to "expected value"